domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/vilkår/Kategori.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.tiltakspenger.saksbehandling.domene.vilkår
```

Comment:
Jeg har forstått det sånn at vi egentlig bare har fire vilkår, og at ‘Vilkår’-enumet vårt egentlig ikke er vilkår. Er det det du løser her? Å samle alt i riktig 'vilkår-gruppe’ (om du forstod spørsmålet 😅)

Reply:
Ja, jeg har også begynt å få den forståelsen.. Kategori-koden har vi hatt hele tiden, men den har vært bortgjemt i en frontend-mapper. Jeg bare dro den ut derfra nå. Men jeg er enig i beskrivelsen din: Det vi har kalt vilkår er egentlig ikke vilkår (men noe annet som jeg ikke har et godt ord for ennå), mens det vi kaller Kategori vel egentlig er Vilkår..